.github/workflows/ci.yml

```diff
@@ -34,9 +34,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        name_suffix: ['']
         php: ['7.3', '7.4', '8.0', '8.1', '8.2']
+        phpunit: [false]
+        include:
+          - name_suffix: with PHPUnit 10.1
```

Comment:
this job should not be added actually. The normal 8.1 and 8.2 jobs are already using PHPUnit 10 as that's the latest compatible stable version

Reply:
Actually, all changes to that file should be reverted